### DISCUSSION TITLE
Multiple result set in a same tab

### DISCUSSION THREAD

**Discussion by MirkoSacco** [dbeaver/dbeaver#36393] [Q&A] [answered]
When I execute multiple queries in the editor, the results are split into different tabs, as shown in the image:
![image](https://github.com/user-attachments/assets/56898840-8d98-4aa6-8f11-f21b20c470cf)

I would like a view similar to SQL Server:
![image](https://github.com/user-attachments/assets/cedae9ed-e81d-47e6-94e1-445b552ef617)

someone can help me?

DBever Version 24.2.5.202411171748

Thank you all!

**Comment by ged-yuko** [comment, marked as answer]
There is a `Show results in a single/multiple tabs` option, which is hidden by default and available in the menus for the current editor (see [this](https://github.com/dbeaver/dbeaver/issues/6861#issuecomment-1873986773) comment). You can also get faster access to it by enabling corresponding toolbar toggle button from the `Toolbar Customizations` preference page, because it is a per-editor's instance option, so its state saved on a per-script basis and disabled by default:

![изображение](https://github.com/user-attachments/assets/ecd54a4c-a17b-4a5c-b28c-0b740996cc25)

**Reply by ged-yuko** [reply]
There is also a description of this feature in the official documentation: https://dbeaver.com/docs/dbeaver/SQL-Editor/#multiple-results-in-one-tab
